donkstation/code/datums/diseases/advance/symptoms/fermentation.dm

```diff
@@ -0,0 +1,101 @@
+/datum/symptom/fermentation
+	name = "Endogenous Ethanol Fermentation"
+	desc = "This symptom causes the gut bacteria of the infected to continually produce ethanol, creating a near constant state of intoxication."
+	stealth = -2
+	resistance = -3
+	stage_speed = -4
+	transmission = 1
+	level = 6
+	severity = 0 				//Entirely harmless besides a waddle at first.
+	symptom_delay_min = 5
+	symptom_delay_max = 7
+	var/ethanol_power = 3.35 	//Level of drunkenness that will be maintained, scales with Transmission & Stage Speed. This also ensures a harmful virus cannot be stealthed.
+	var/drunken_healing = FALSE //Grants Drunken Resilience with a threshold
+	var/has_light_drinker = FALSE //storage for existing traits
+	var/has_drunk_healing = FALSE //^^^^^
+	var/datum/component/is_waddling = FALSE		//For the waddle trait.
+	threshold_desc = "<b>Transmission & Stage Speed:</b> The maximum amount of alcohol in the system scales based off the transmission and stage speed stat.<br>\
+					<b>Resistance 8:</b> The symptom now heals the infected while drunk and makes light drinkers more capable of holding their liquor. Has no effect on those that are already resilient to alcohol."
+
+/datum/symptom/fermentation/severityset(datum/disease/advance/A)
+	ethanol_power = (A.transmission * A.stage_rate + 3)
```

Comment:
A minimum of 0 would cause the disease to have absolutely zero effect, sans the messages.
In the case of the different math, the 3 in the severityset is because there is no need to add that level of precision to the check. 
You need 3.35 as a baseline in the actual disease itself to allow for the Ballmer Effect (13.35 drunkenness) that scientists can get.